weblm/controller.py

```diff
@@ -212,7 +214,7 @@ def reset_state(self):
     def success(self):
         for state, command in self.moments:
             self._save_example(state, command)
-
+    
```

Comment:
can you use `yapf --style "{based_on_style: google, indent_width: 4, column_limit: 120}" -ir .` to format this repo?

Reply:
Sure, Sorry about the lack of proper formatting before. I formatted and pushed the changes